ansible/playbook/tasks/roles/sqlite-to-postgres/tasks/main.yml

```diff
@@ -103,3 +112,8 @@
   file:
     path: /srv/grafana/grafana.db
     state: absent
+
+- name: Start nginx again
+  supervisorctl:
+    name: nginx
+    state: start
```

Comment:
grafana started later there: https://github.com/percona/pmm-update/blob/86496338d91e4637ca2beb5d94c2a5a8cc8a091b/ansible/playbook/tasks/update.yml#L249 ?

Reply:
Yes, because if we don't start grafana here won't be able to see upgrade progress in UI.